easycv/hooks/tensorboard.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+import numpy as np
+import torch
+from mmcv.runner.dist_utils import master_only
+from mmcv.runner.hooks import HOOKS
+from mmcv.runner.hooks import TensorboardLoggerHook as _TensorboardLoggerHook
+
+
+@HOOKS.register_module()
+class TensorboardLoggerHookV2(_TensorboardLoggerHook):
+
+    def visualization_log(self, runner):
+        visual_results = runner.visualization_buffer.output
+        for vis_key, vis_result in visual_results.items():
+            images = vis_result.get('images', [])
+            img_metas = vis_result.get('img_metas', [])
+            assert len(images) == len(
+                img_metas
+            ), 'Output `images` and `img_metas` must keep the same length!'
+
+            for i, img in enumerate(images):
+                if isinstance(img, np.ndarray):
+                    img = torch.from_numpy(img)
+                else:
+                    assert isinstance(
+                        img, torch.Tensor
+                    ), 'Only support np.ndarray and torch.Tensor type!'
+
+                filename = img_metas[i]['ori_filename']
```

Comment:
is filename sufficient for visualization?   img_metas is not straightforward

Reply:
add doc, img_metas is optional

Reply:
if imag_metas does not exsits in vis_result, line 29 will throw exception